assets/app/scripts/services/metrics.js

```diff
@@ -69,7 +74,9 @@ angular.module("openshiftConsole")
           // Save the raw value before we change it.
           value = point.value;
           point.value = millicoresUsed(point, lastValue);
-          lastValue = value;
+          if (value) {
+            lastValue = value;
```

Comment:
just checking, should you be setting lastValue to undefined/null if ! value?  or are you intentionally comparing to a potentially older lastValue